pkgs/applications/virtualization/kvmtool/default.nix

```
@@ -1,12 +1,13 @@
-{ stdenv, fetchzip, lib, dtc }:
+{ stdenv, fetchgit, lib, dtc }:
 
 stdenv.mkDerivation {
   pname = "kvmtool";
-  version = "unstable-2023-07-12";
+  version = "unstable-2024-06-14";
 
-  src = fetchzip {
-    url = "https://git.kernel.org/pub/scm/linux/kernel/git/will/kvmtool.git/snapshot/kvmtool-106e2ea7756d980454d68631b87d5e25ba4e4881.tar.gz";
-    hash = "sha256-wpc5DfHnui0lBVH4uOq6a7pXVUZStjNLRvauu6QpRvE=";
+  src = fetchgit {
+    url = "https://github.com/kvmtool/kvmtool.git";
+    rev = "da4cfc3e540341b84c4bbad705b5a15865bc1f80";
+    hash = "sha256-05tNsZauOXe1L1y1YchzvLZm3xOctPJhHCjyAyRnwy4=";
   };
```

maybe it would be better to use the github specific fetcher? remember to update the hash and the inputs.

```suggestion
  src = fetchFromGitHub {
    owner = "kvmtool";
    repo = "kvmtool";
    rev = "da4cfc3e540341b84c4bbad705b5a15865bc1f80";
    hash = "sha256-?????";
  };
```